### ISSUE TITLE
Exception when placing blocks above the height limit

### ISSUE THREAD

**Issue by TogAr2** [Minestom/Minestom#867]
Placing a block above the dimension height limit causes an exception:
```
java.lang.ArrayIndexOutOfBoundsException: Index 24 out of bounds for length 24
	at java.base/java.util.ImmutableCollections$ListN.get(ImmutableCollections.java:680)
	at net.minestom.server.instance.DynamicChunk.getSection(DynamicChunk.java:109)
	at net.minestom.server.instance.Chunk.getSectionAt(Chunk.java:93)
	at net.minestom.server.instance.DynamicChunk.setBlock(DynamicChunk.java:71)
	at net.minestom.server.instance.InstanceContainer.UNSAFE_setBlock(InstanceContainer.java:131)
	at net.minestom.server.instance.InstanceContainer.placeBlock(InstanceContainer.java:166)
	at net.minestom.server.listener.BlockPlacementListener.listener(BlockPlacementListener.java:133)
	at net.minestom.server.listener.manager.PacketListenerManager.processClientPacket(PacketListenerManager.java:88)
```

This should be easy to fix by checking if the placed block is within the world limits inside of `BlockPlacementListener`.

**Comment by Nuckerr** [comment]
Hi, I was attempting this, how do I get the Y value of the block in BlockPlacementListener. I got blockPosition.y() however that returns the y value of the block it was placed off.

**Comment by mworzala** [comment]
Fixed with minestom-ce in #1937
